### PR TITLE
changes made to models/engine/db_storage.py

### PR DESCRIPTION
**Pull request by benjminrichie**
changes made to models/engine/file_storage.py

### CODE REVIEW

**Comment by benjminrichie** [comment]
@yomex96 Kindly review the PR.

**Comment by Yomex1996** [comment]
Good work

File: README.md [hunk 1]
```diff
@@ -156,6 +156,8 @@ No known bugs at this time.
 ## Authors
 Alexa Orrico - [Github](https://github.com/alexaorrico) / [Twitter](https://twitter.com/alexa_orrico)  
 Jennifer Huang - [Github](https://github.com/jhuang10123) / [Twitter](https://twitter.com/earthtojhuang)
+Benjamin Richard - [Github](https://github.com/benjminrichie) / [Twitter](https://twitter.com/benjminrichie)
+Abayomi Robert Onawole - [Github](https://github.com/yomex96) / [Twitter]()
```
Comment (Yomex1996) [inline README.md]: good work

**Review by Yomex1996 — APPROVED** [review]
great work

**Comment by Yomex1996** [comment]
nice

**Review by benjminrichie — COMMENTED** [review]
@yomex96  @Yomex1996  please review

**Review by Yomex1996 — APPROVED** [review]

**Pull request closed without merging**